bindings/rust/wasmedge-sys/src/vm.rs

```diff
@@ -532,7 +592,7 @@ impl Vm {
     /// # Error
     ///
     /// If fail to load, then an error is returned.  The loaded module is not validated.
-    pub fn load_wasm_from_bytes(&mut self, bytes: &[u8]) -> WasmEdgeResult<()> {
+    pub fn load_wasm_from_bytes(&self, bytes: &[u8]) -> WasmEdgeResult<()> {
```

Comment:
Please still use '&mut self` as the design of the current `sys` APIs tried to "express" the mutability of the relevant C-API.